forum/profiles/serializers.py

```diff
@@ -83,3 +83,35 @@ class Meta:
             'city',
             'description',
         ]
+
+
+class ViewedStratupReadSerializer(serializers.ModelSerializer):
+    
+    startup_id= serializers.StringRelatedField(source='startup.id', read_only=True)
+    company_name= serializers.StringRelatedField(source='startup.company_name', read_only=True)
```

Comment:
Instead of `StringRelatedField`, use `SlugRelatedField` for more flexibility. This allows you to specify which field of the related model to display.
```
startup_id = serializers.SlugRelatedField(source='startup', slug_field='id', read_only=True)
company_name = serializers.SlugRelatedField(source='startup', slug_field='company_name', read_only=True)
```